packages/testcontainers/src/chains/reg_test_container/spv.ts

```diff
@@ -0,0 +1,31 @@
+import { RegTestContainer } from './'
+
+/**
+ * SPV testing functionnalities
+ */
+export class Spv {
+  private readonly client: RegTestContainer
+
+  constructor (client: RegTestContainer) {
+    this.client = client
+  }
+
+  /**
+   * Funds a Bitcoin address with 1 BTC.
```

Comment:
hmm.. do we hardcode the funding amount?? @surangap  :thinking:

Reply:
looks like it.
C++ side https://github.com/DeFiCh/ain/blob/1cdc7e05b8340fbd51ba76a1263a6e1fb194e6cd/src/spv/spv_wrapper.cpp#L1586